model_compression_toolkit/exporter/target_platform_export/keras/exporters/fakely_quant_keras_exporter.py

```diff
@@ -0,0 +1,124 @@
+    NoOpQuantizeConfig
+
+from model_compression_toolkit.core.common import Logger
+from model_compression_toolkit.exporter.fully_quantized.keras.builder.quantize_config_to_node import \
+    SUPPORTED_QUANTIZATION_CONFIG
+from model_compression_toolkit.exporter.fully_quantized.keras.quantize_configs.weights_quantize_config import \
+    WeightsQuantizeConfig
+from model_compression_toolkit.exporter.target_platform_export.keras.exporters.base_keras_exporter import BaseKerasExporter
+
+
+class FakelyQuantKerasExporter(BaseKerasExporter):
+    """
+    Exporter for fakely-quant Keras models.
+    """
+
+    def __init__(self, model:keras.models.Model):
+        super().__init__(model)
+
+    def export(self) -> keras.models.Model:
+        """
+        Convert fully-quantized Keras model to a fakely-quant export-ready model.
+
+        Returns:
+            Fake-quant Keras model ready to export.
+        """
+
+        def _unwrap_quantize_wrapper(layer:Layer):
+            """
+            Convert layer wrapped with QuantizeWrapper to the layer it wraps, so it's ready to export.
+
+            Args:
+                layer: Layer to unwrap.
+
+            Returns:
+                Layer after unwrapping.
+
+            """
+
+            if isinstance(layer, tfmot.quantization.keras.QuantizeWrapper):
+                # For now, we assume only weights are quantized for models to export. Activation support is in progress.
+                if isinstance(layer.layer, InputLayer):
+                    if not isinstance(layer.quantize_config, NoOpQuantizeConfig):
+                        Logger.error(
+                            f'Currently, FakelyQuantKerasExporter supports weights quantization only. Please consider '
+                            f'disabling activation quantization, until supported ')
+                    return Layer() #Identity
+
+                if not isinstance(layer.quantize_config, tuple(SUPPORTED_QUANTIZATION_CONFIG)):
+                    Logger.error(
+                        f'Only supported quantization configs are {tuple(SUPPORTED_QUANTIZATION_CONFIG)}')
+
+                # If weights are quantized, use the quantized weight for the new built layer.
+                if isinstance(layer.quantize_config, WeightsQuantizeConfig):
+                    new_layer = layer.layer.__class__.from_config(layer.layer.get_config())
+                    with tf.name_scope(new_layer.name):
+                        new_layer.build(layer.input_shape)
+
+                    # Build a list of the layer's new weights.
+                    weights_list = []
+                    for w in new_layer.weights:
+                        val = None
+                        for qw in layer.weights:
+                            if w.name in qw.name:
+                                if w.name.split('/')[-1].split(':')[0] in layer.quantize_config.weight_attrs:
```

Comment:
Maybe add a comment for what is the expected patters here so it would be easier to debug in case of failure